src/state/lottery/getLotteriesData.ts

```diff
@@ -3,7 +3,10 @@ import { GRAPH_API_LOTTERY } from 'config/constants/endpoints'
 import { LotteryRoundGraphEntity, LotteryResponse } from 'state/types'
 import { getRoundIdsArray, fetchMultipleLotteries } from './helpers'
 
-const MAX_LOTTERIES_REQUEST_SIZE = 100
+export const MAX_LOTTERIES_REQUEST_SIZE = 100
+
+/* eslint-disable camelcase */
+type LotteriesWhere = { id_in?: string[] }
```

Comment:
Putting this here as a reminder to test it out when we have a chance. We might be able to simply generate the types automatically based on the graph.
https://www.graphql-code-generator.com/